desktop-app/package.json

```diff
@@ -27,10 +27,10 @@
     "build": "concurrently \"yarn run build:main\" \"yarn run build:renderer\"",
     "build:main": "cross-env NODE_ENV=production TS_NODE_TRANSPILE_ONLY=true webpack --config ./.erb/configs/webpack.config.main.prod.ts",
     "build:renderer": "cross-env NODE_ENV=production TS_NODE_TRANSPILE_ONLY=true webpack --config ./.erb/configs/webpack.config.renderer.prod.ts",
-    "postinstall": "yarn rimraf --glob node_modules/browser-sync/dist/**/*.map && yarn replace '\"network-throttle\".*' '' node_modules/browser-sync-ui/lib/UI.js && ts-node .erb/scripts/check-native-dep.js && replace-in-file '/// <reference types=\"howler\" />' \"import { Howl } from 'howler';\" node_modules/use-sound/dist/types.d.ts  && electron-builder install-app-deps && cross-env NODE_ENV=development TS_NODE_TRANSPILE_ONLY=true webpack --config ./.erb/configs/webpack.config.renderer.dev.dll.ts",
+    "postinstall": "ts-node postinstall.ts",
```

Comment:
Can we keep the app dependencies installation and webpack here only? 
So that the `postinstall.ts` can only take care of the housekeeping stuff and at the same time the visibility of these important commands is maintained too.

Reply:
Yes.

Reply:
The updated commit seems to have added a duplicate entry for `postinstall`. Hope that's not intentional.

Reply:
Hi Manoj,
Thank you so much for the feedback. Please consider the latest changes.

Reply:
Looks good now, thank you for your patience!

Reply:
Will merge this after cutting a release. So, we'll have time to test and fix if there are any issues.